README.md

```diff
@@ -1,12 +1,53 @@
-# Northcoders News API
+# Welcome to my News API
+### Introduction
+Thank you for taking the time to view my News API. This portfolio project was made during a week-long sprint (26-8-2024 to 30-8-2024) during my [Northcoders](https://northcoders.com/) full stack software developer bootcamp. During this week I utilised the following skills in order to finish this project:
+* Javascript
+* Documentation
+* Test driven development (TDD)
+* Jest
+* Supertest
+* Express
+* Model view controller (MVC)
+* PostgreSQL
+* Fs/promises
+* Hosting using supabase for the database and render for the website
```

Comment:
I would focus more on what the project actually is - you're talking about the concepts at play but you haven't actually said what the project is for or does